modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/internal/util/DDMXMLImpl.java

```diff
@@ -73,7 +73,7 @@ public Fields getFields(
 			List<String> fieldNames)
 		throws PortalException {
 
-		Document document = null;
+		Document document;
```

Comment:
@JanisZhang Was this change required by Source Formatter?

Reply:
@dantewang Yes

Reply:
@dantewang No, It's recommended by IDE.

Reply:
@JanisZhang Please ignore the IDE recommendation. Actually it's a convention in Liferay to give a variable an initial value, even if it's not used.

Reply:
@dantewang Ok